packages/admin-ui/src/filters/multiselect-filter.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import { tag } from '@vtex/admin-ui-react'
+import { Checkbox } from '../components/Checkbox'
+import { Filter } from './filter'
+import { Option } from './option'
+import type { FilterItem, UseMultipleFilterReturn } from './filter.state'
+
+export function MultiselectFilter(props: MultiselectFilterProps) {
+  const {
+    state: { listState, appliedValues },
```

Comment:
This prevents appliedValues from being undefined

```suggestion
    state: { listState, appliedValues = [] },
```